server/server.go

```diff
@@ -81,16 +115,22 @@ func (s *Server) Serve() error {
 		UseAuth:            s.useAuth(),
 	}, service)
 
+	// Add chronograf's version header to all requests
 	s.handler = Version(s.BuildInfo.Version, s.handler)
 
-	var err error
-	s.Listener, err = net.Listen("tcp", net.JoinHostPort(s.Host, strconv.Itoa(s.Port)))
+	if s.useTLS() {
+		// Add HSTS to instruct all browsers to change from http to https
+		s.handler = HSTS(s.handler)
```

Comment:
Great, will non browser clients honor the HSTS headers or will you also need to create a redirect from http to https for those clients? I know nothing about HSTS so just asking the questions hoping you know the answers.